tests/test-td-payload/src/testtdinfo.rs

```diff
@@ -48,11 +48,10 @@ impl TestCase for Tdinfo {
      */
     fn run(&mut self) {
         let mut td_info = tdx::tdcall_get_td_info().expect("Failt to get td info");
-
-        if (self.expected.gpaw != td_info.gpaw) {
```

Comment:
please remove expected.gpaw, if it is not needed.

Reply:
Resolved